package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "cypress-cucumber-attach-screenshots-to-failed-steps",
-  "version": "1.0.0",
+  "version": "1.1.0",
```

Comment:
If we are going to change the API we need this to be a major version bump and we will also need to probably publish to npm which I had been lazy and didn't do.

Reply:
Fair point, I agree.
I've change the version to 2